api/broker-client.yaml

```diff
@@ -1511,6 +1545,29 @@ components:
       type: array
       items:
         $ref: '#/components/schemas/Subscription'
+    Notification:
```

Comment:
Would be good to set the required properties, e.g. id, type, subscriptionId, notfiedAt, data. Beside that, could you add examples for each value? This will allow us to generate tests in the future.

Reply:
I wanted to add examples and I checked how it's done in other schemas. I see something like this: 'externalValue: 'http://localhost:8090/examples/Subscription-example_C.4.json''
Could you tell me, where are these examples located?

Reply:
Got it, done